engine/table/src/test/java/io/deephaven/engine/util/scripts/TestGroovyDeephavenSession.java

```diff
@@ -104,5 +136,497 @@ public void testUnloadedWildcardPackageImport() {
+            session.evaluateScript(c).throwIfError();
+            Double wrapperMax = (Double) session.getVariable("m2");
+            assertEquals(5.0d, wrapperMax, 0.0d);
+        } catch (Exception e) {
+            e.printStackTrace();
+            fail("Fail for : \n" + c);
+        }
+
+        c = "t = emptyTable(1).updateView(\"Y=" + Y + "\", \"Z=min(Y, z)\")\n";
+        try {
+            QueryScope.addParam("z", z);
+            session.evaluateScript(c).throwIfError();
+            final Table t = (Table) session.getVariable("t");
+            final int var2 = t.getColumnSource("Z").getInt(0);
+            assertEquals(Numeric.min(Y, z), var2);
+        } catch (Exception e) {
+            e.printStackTrace();
+            fail("Fail for : \n" + c);
+        }
+
+        c = "t = emptyTable(1).updateView(\"Y=" + Y + "\", \"Z=min(Y, 5d)\")\n";
+        try {
+            QueryScope.addParam("z", z);
+            session.evaluateScript(c).throwIfError();
+            final Table t = (Table) session.getVariable("t");
+            final double var2 = t.getColumnSource("Z").getDouble(0);
+            assertEquals(Numeric.min(Y, 5d), var2, 1e-10);
+        } catch (Exception e) {
+            e.printStackTrace();
+            fail("Fail for : \n" + c);
+        }
+
+        c = "t = emptyTable(1).updateView(\"Y=" + Y + "\", \"Z=min(Y, d)\")\n";
+        try {
+            QueryScope.addParam("z", z);
+            QueryScope.addParam("d", d);
+            session.evaluateScript(c).throwIfError();
+            final Table t = (Table) session.getVariable("t");
+            final double var2 = t.getColumnSource("Z").getDouble(0);
+            assertEquals(Numeric.min(Y, d), var2, 1e-10);
+        } catch (Exception e) {
+            e.printStackTrace();
+            fail("Fail for : \n" + c);
+        }
+
+        c = "t = emptyTable(1).updateView(\"Y=" + Y + "\", \"Z=max(Y, z)\")\n";
+        try {
+            QueryScope.addParam("z", z);
+            session.evaluateScript(c).throwIfError();
+            final Table t = (Table) session.getVariable("t");
+            final int var2 = t.getColumnSource("Z").getInt(0);
+            assertEquals(Numeric.max(Y, z), var2);
+        } catch (Exception e) {
+            e.printStackTrace();
+            fail("Fail for : \n" + c);
+        }
+
+
+        c = "t = emptyTable(1).updateView(\"Y=" + Y + "\", \"Z=max(Y, 5d)\")\n";
+        try {
+            QueryScope.addParam("z", z);
+            session.evaluateScript(c).throwIfError();
+            final Table t = (Table) session.getVariable("t");
+            final double var2 = t.getColumnSource("Z").getDouble(0);
+            assertEquals(Numeric.max(Y, 5d), var2, 1e-10);
+        } catch (Exception e) {
+            e.printStackTrace();
+            fail("Fail for : \n" + c);
+        }
+
+        c = "t = emptyTable(1).updateView(\"Y=" + Y + "\", \"Z=max(Y, d)\")\n";
+        try {
+            QueryScope.addParam("z", z);
```

Comment:
I don't understand why you're `addParam` on each test - in this particular case you're setting `z` but looks unused. Why not set all parameters once at the top of the method?

Reply:
Agreed, this seems a bit off, but is part of the original test from DHE. I tried to keep my changes here limited, ensuring that except where I explicitly changed something, the same things that pass for DHE passes in DHC.